[2022-05-23 02:46:57,643] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: anp_fuel_sales_etl.insert_raw_sales_of_oil_derivative_fuels manual__2022-05-23T02:46:53.394080+00:00 [queued]>
[2022-05-23 02:46:57,650] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: anp_fuel_sales_etl.insert_raw_sales_of_oil_derivative_fuels manual__2022-05-23T02:46:53.394080+00:00 [queued]>
[2022-05-23 02:46:57,650] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 02:46:57,650] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-05-23 02:46:57,650] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 02:46:57,763] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): insert_raw_sales_of_oil_derivative_fuels> on 2022-05-23 02:46:53.394080+00:00
[2022-05-23 02:46:57,768] {standard_task_runner.py:52} INFO - Started process 279 to run task
[2022-05-23 02:46:57,769] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'anp_fuel_sales_etl', 'insert_raw_sales_of_oil_derivative_fuels', 'manual__2022-05-23T02:46:53.394080+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/anp_fuel_sales_etl.py', '--cfg-path', '/tmp/tmpvb1whbr_', '--error-file', '/tmp/tmpicks5ipn']
[2022-05-23 02:46:57,770] {standard_task_runner.py:80} INFO - Job 10: Subtask insert_raw_sales_of_oil_derivative_fuels
[2022-05-23 02:46:57,805] {task_command.py:369} INFO - Running <TaskInstance: anp_fuel_sales_etl.insert_raw_sales_of_oil_derivative_fuels manual__2022-05-23T02:46:53.394080+00:00 [running]> on host f60453fe3eb1
[2022-05-23 02:46:57,862] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=anp_fuel_sales_etl
AIRFLOW_CTX_TASK_ID=insert_raw_sales_of_oil_derivative_fuels
AIRFLOW_CTX_EXECUTION_DATE=2022-05-23T02:46:53.394080+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-23T02:46:53.394080+00:00
[2022-05-23 02:46:57,863] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-05-23 02:46:57,863] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-05-23 02:46:57,865] {dbapi.py:208} INFO - Running statement: INSERT INTO public.raw_sales_of_oil_derivative_fuels
select 
        cast(ano || '-' || unnest(array['01' , '02' , '03' , '04' , '05' , '06' , '07' , '08' , '09' , '10' , '11' , '12']) || '-' || '01' as date) as year_month,
        estado as uf,
        combustivel as product,
        regiao as unit,
        unnest(array[jan , fev , mar , abr , mai, jun, jul, ago, set, out, nov, dez]) as volume,
        CURRENT_TIMESTAMP as created_at 
from public.sales_of_oil_derivative_fuels;

TRUNCATE TABLE public.sales_of_oil_derivative_fuels;, parameters: None
[2022-05-23 02:46:57,867] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 188, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 212, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedTable: relation "public.raw_sales_of_oil_derivative_fuels" does not exist
LINE 1: INSERT INTO public.raw_sales_of_oil_derivative_fuels
                    ^

[2022-05-23 02:46:57,873] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=anp_fuel_sales_etl, task_id=insert_raw_sales_of_oil_derivative_fuels, execution_date=20220523T024653, start_date=20220523T024657, end_date=20220523T024657
[2022-05-23 02:46:57,883] {standard_task_runner.py:97} ERROR - Failed to execute job 10 for task insert_raw_sales_of_oil_derivative_fuels (relation "public.raw_sales_of_oil_derivative_fuels" does not exist
LINE 1: INSERT INTO public.raw_sales_of_oil_derivative_fuels
                    ^
; 279)
[2022-05-23 02:46:57,901] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-05-23 02:46:57,932] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
